mne/preprocessing/eog.py

```diff
@@ -49,15 +49,19 @@ def find_eog_events(raw, event_id=998, l_freq=1, h_freq=10,
     eog_inds = _get_eog_channel_index(ch_name, raw)
     logger.info('EOG channel index for this subject is: %s' % eog_inds)
 
-    eog, _ = raw[eog_inds, :]
+    # Reject bad segments.
+    eog, times = raw.get_data(picks=eog_inds, reject_by_annotation='omit',
```

Comment:
It seems like the approach here is better, so my vote would be to unify them.

It just occurred to me that after finding events, you don't want to keep any that are too close to a border, because then you're taking data from an unrelated segment